lib/pbench/cli/server/tree_manage.py

```diff
@@ -37,13 +37,27 @@ def print_tree(tree: FileTree):
                     print(f"        States: {', '.join(states)}")
 
 
-@click.command(name="pbench-tree-manager")
+@click.group("pbench-tree-manage")
```

Comment:
What about the removal of `pbench-server-prep-shim-002` required these changes below? Could this be "leaked" changes from another PR?